README.md

```diff
@@ -8,6 +8,7 @@ This fork is based on the 1.x version rather than newer 2.x version.  We made th
 - Adds support for constant tables using `VALUES` lists.
 - Adds support for `LATERAL` joins.
 - Fix identifier quoting. Old code wasn't quoting identifiers in all cases: https://github.com/Ff00ff/mammoth/issues/576
+- Adds ability to get a column's definition from a column.
```

Comment:
This sentence is confusing.

One option is to be more precise, e.g.
> `Column` objects now contain a reference to their `ColumnDefinition`.

But I think in this case we maybe don't need a changelog entry at all. It's a relatively niche thing that I think is fine for people to discover whenever they happen to discover it?

Reply:
I changed the sentence to use your suggestion.